[2025-02-06T17:03:34.113+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T17:03:34.135+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_online_store_data manual__2025-02-06T16:33:25.405424+00:00 [queued]>
[2025-02-06T17:03:34.143+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_online_store_data manual__2025-02-06T16:33:25.405424+00:00 [queued]>
[2025-02-06T17:03:34.144+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 4
[2025-02-06T17:03:34.160+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_online_store_data> on 2025-02-06 16:33:25.405424+00:00
[2025-02-06T17:03:34.174+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=242) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-06T17:03:34.175+0000] {standard_task_runner.py:72} INFO - Started process 246 to run task
[2025-02-06T17:03:34.176+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag_incremental', 'extract_online_store_data', 'manual__2025-02-06T16:33:25.405424+00:00', '--job-id', '412', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp3z_njj9n']
[2025-02-06T17:03:34.177+0000] {standard_task_runner.py:105} INFO - Job 412: Subtask extract_online_store_data
[2025-02-06T17:03:34.230+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag_incremental.extract_online_store_data manual__2025-02-06T16:33:25.405424+00:00 [running]> on host f34e19f1f8c5
[2025-02-06T17:03:34.341+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag_incremental' AIRFLOW_CTX_TASK_ID='extract_online_store_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T16:33:25.405424+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-06T16:33:25.405424+00:00'
[2025-02-06T17:03:34.342+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-06T17:03:34.343+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-06T17:03:34.343+0000] {logging_mixin.py:190} INFO - Current task name:extract_online_store_data state:running start_date:2025-02-06 17:03:34.135740+00:00
[2025-02-06T17:03:34.343+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental and current dag run status:running
[2025-02-06T17:03:34.343+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T17:03:34.373+0000] {base.py:84} INFO - Retrieving connection 'online_store_conn'
[2025-02-06T17:03:34.377+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM online_orders 
            WHERE created_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-06T17:03:34.391+0000] {sql.py:562} INFO - Rows affected: 6558
[2025-02-06T17:03:34.641+0000] {etl.py:150} INFO - Extracted orders 6558 new rows from db_online_store.
[2025-02-06T17:03:34.654+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM customers 
            WHERE updated_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-06T17:03:34.663+0000] {sql.py:562} INFO - Rows affected: 3301
[2025-02-06T17:03:34.756+0000] {etl.py:170} INFO - Extracted customers 3301 new rows from db_online_store.
[2025-02-06T17:03:34.764+0000] {python.py:240} INFO - Done. Returned value was: ('/tmp/online_oders_data.json', '/tmp/online_customer_data.json')
[2025-02-06T17:03:34.822+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T17:03:34.823+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag_incremental, task_id=extract_online_store_data, run_id=manual__2025-02-06T16:33:25.405424+00:00, execution_date=20250206T163325, start_date=20250206T170334, end_date=20250206T170334
[2025-02-06T17:03:34.842+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-06T17:03:34.842+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-06T17:03:34.843+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental queued_at:2025-02-06 17:00:10.527378+00:00
[2025-02-06T17:03:34.843+0000] {logging_mixin.py:190} INFO - Task hostname:f34e19f1f8c5 operator:PythonOperator
[2025-02-06T17:03:34.880+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-06T17:03:34.935+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-06T17:03:34.937+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
